test/uk/gov/hmrc/transitmovementsvalidator/data/cc015c-generated-by-converter.json

```diff
@@ -0,0 +1,73 @@
+{
+  "n1:CC015C": {
+    "messageSender": "test",
```

Comment:
This wasn't actually meant to catch anyone out (honest!), but I just realised the fact this is here means that the JSON schema in the validator are wrong - the `messageSender` needs to be removed.